cellprofiler/modules/savecroppedobjects.py

```diff
@@ -25,27 +25,54 @@
 import numpy
 import os.path
 import skimage.io
+import skimage.measure
 import time
 
 import cellprofiler.module
 import cellprofiler.setting
 
 O_PNG = "png"
 O_TIFF = "tiff"
+SAVE_PER_OBJECT = "Images"
+SAVE_MASK = "Masks"
 
 class SaveCroppedObjects(cellprofiler.module.Module):
     category = "File Processing"
 
     module_name = "SaveCroppedObjects"
 
-    variable_revision_number = 1
+    variable_revision_number = 2
 
     def create_settings(self):
+        self.export_option = cellprofiler.setting.Choice(
+            "Do you want to save cropped images or object masks?",
+            [
+                SAVE_PER_OBJECT,
+                SAVE_MASK
+            ],
+            doc="""
+            Choose the way you want the per-object crops to be exported.
+            <p>The choices are:<br>
+            <ul><li><i>{SAVE_PER_OBJECT}</i>: Save a per-object crop from the original image based on the object's
+            bounding box.</li>
+            <li><i>{SAVE_MASK}</i>: Export a per-object mask.</li>
+            </ul></p>
+            """.format(**{
+                "SAVE_PER_OBJECT": SAVE_PER_OBJECT,
```

Comment:
I know this is how other modules do it, but for IDE ease do you mind replacing this with the following?
```
.format(
    SAVE_PER_OBJECT=SAVE_PER_OBJECT,
    SAVE_MASK=SAVE_MASK
)
```